### PR TITLE
Update azure-cli-extension-alias.md

### PR DESCRIPTION
**Pull request by alexandair**
Fix typos.

### CODE REVIEW

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/azure-cli-extension-alias.md](https://github.com/alexandair/azure-docs-cli-python/blob/patch-3/docs-ref-conceptual/azure-cli-extension-alias.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-extension-alias?view=azure-cli-latest&branch=pr-en-us-717) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-extension-alias?view=azure-cli-2017-03-09-profile&branch=pr-en-us-717) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C3%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201804032228590492-717%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vyrGbjxn9%2BWuEYBrRmI60342JFRwxdQtyw4okDeZvgg%3D&st=2018-04-03T22%3A26%3A45Z&se=2018-05-04T22%3A31%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/azure-cli-extension-alias.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ az ls-groups
 
 ## Create an alias command with arguments
 
-You can also add positional arguments to an alias command by including them as `{{ arg_name }}` in the alias name. The whitespace inside the curly braces is required.
+You can also add positional arguments to an alias command by including them as `{{ arg_name }}` in the alias name. The whitespace inside the curly brackets is required.
```
Comment (sptramer) [inline docs-ref-conceptual/azure-cli-extension-alias.md]: Remove change; `curly braces` is standard US-EN nomenclature for `{}`.

Comment (alexandair) [reply docs-ref-conceptual/azure-cli-extension-alias.md]: Shouldn't that be just "braces" then? All braces are curly.

Comment (sptramer) [reply docs-ref-conceptual/azure-cli-extension-alias.md]: Seems fine - go ahead and change to `braces`.

Comment (alexandair) [reply docs-ref-conceptual/azure-cli-extension-alias.md]: Changed.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/azure-cli-extension-alias.md](https://github.com/alexandair/azure-docs-cli-python/blob/patch-3/docs-ref-conceptual/azure-cli-extension-alias.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-extension-alias?view=azure-cli-latest&branch=pr-en-us-717) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-extension-alias?view=azure-cli-2017-03-09-profile&branch=pr-en-us-717) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C4%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201804041805465227-717%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=7PZK84PwXznQ7PurWJ2fbU%2BIJhU4XjG7sgFhyT%2BhBbs%3D&st=2018-04-04T18%3A03%3A33Z&se=2018-05-05T18%3A08%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sptramer** [comment]
Thanks for your contribution!

**Pull request merged**